src/main/java/io/supertokens/session/Session.java

```diff
@@ -145,7 +145,7 @@ public static SessionInformationHolder regenerateToken(Main main, @Nonnull Strin
 
     // pass antiCsrfToken to disable csrf check for this request
     public static SessionInformationHolder getSession(Main main, @Nonnull String token, @Nullable String antiCsrfToken,
-            boolean enableAntiCsrf, Boolean doAntiCsrfCheck) throws StorageQueryException,
+            boolean enableAntiCsrf, Boolean doAntiCsrfCheck, Boolean checkDatabase) throws StorageQueryException,
```

Comment:
why not boolean and why Boolean? Boolean makes it nullable

Reply:
I was following `doAntiCsrfCheck`, but I'll update it.